src/backend/metal/InputStateMTL.mm

```diff
@@ -30,6 +30,12 @@ MTLVertexFormat VertexFormatType(nxt::VertexFormat format) {
                     return MTLVertexFormatFloat2;
                 case nxt::VertexFormat::FloatR32:
                     return MTLVertexFormatFloat;
+                case nxt::VertexFormat::UnormR8G8B8A8:
+                    return MTLVertexFormatUChar4Normalized;
+                case nxt::VertexFormat::UnormR8G8:
+                    return MTLVertexFormatUChar2Normalized;
+                case nxt::VertexFormat::UnormR8:
+                    return MTLVertexFormatUCharNormalized;
```

Comment:
Looks like this is only available on more recent Apple systems, can we skip UnormR8 for now? Sorry for the churn.